Selenium.HtmlElements/HtmlElements.csproj

```diff
@@ -1,60 +1,69 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFrameworks>net40;netstandard2.0;</TargetFrameworks>
-    <OutputType>Library</OutputType>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-  </PropertyGroup>
-  <PropertyGroup>
-    <PackageId>HtmlElements</PackageId>
-    <PackageVersion>3.2.0</PackageVersion>
-    <Authors>eger</Authors>
-    <PackageTags>Selenium WebDriver</PackageTags>
-    <PackageLicenseExpression>GPL-3.0-or-later</PackageLicenseExpression>
-    <PackageProjectUrl>http://github.com/eger-geger/Selenium.HtmlElements.Net</PackageProjectUrl>
-    <PackageReleaseNotes>
-      3.2.0:
-      - target netstandard
-      3.1.0:
-      - deprecate `SwitchToLastOpenedWindow` extension method
-      - update methods waiting for opening tabs to return opened window handles
-      3.0.1:
-      - update selenium references to 3.7.0
-      - update 'PreviousSibling', 'NextSibling', 'FirstChild', 'LastChild' properties to return only element nodes
-      3.0.0:
-      - update selenium reference to 3.5.2
-      - add NextSibling WebElement property
-      2.0.2:
-      - handle InvalidOperationException thrown by IOS when element was not found in cache
-      2.0.1:
-      - unwrap raw WebElement from multilayered wrapper during FrameContextOverride initialization
-      2.0.0:
-      - page object factory rewritten from scratches: now it has interface, default implementation and better documentation
-      - redesing element groups and removed extension method used for getting elements from a group
-      - added page factory property to custom elements
-      - added documentation for most public members
-    </PackageReleaseNotes>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DocumentationFile>bin\Debug\HtmlElements.XML</DocumentationFile>
-    <LangVersion>default</LangVersion>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DocumentationFile>bin\Release\HtmlElements.XML</DocumentationFile>
-  </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="Selenium.Support" Version="3.7.0" />
-    <PackageReference Include="Selenium.WebDriver" Version="3.7.0" />
-  </ItemGroup>
-  <Target Name="nuget-package" DependsOnTargets="Build">
-    <Exec Command="nuget spec" />
-    <Exec Command="nuget pack -o $(OutputPath)" />
-    <Exec Command="nuget pack -s -o $(OutputPath)" />
-  </Target>
-  <Target Name="BeforeBuild">
-    <ItemGroup Label="Nupkg">
-      <nupkg Include="**/*.nupkg" />
-    </ItemGroup>
-    <Delete Files="@(nupkg)" />
-  </Target>
-  <Target Name="AfterBuild"></Target>
-</Project>
\ No newline at end of file
+	<PropertyGroup>
+		<TargetFrameworks>net45;net5.0;netstandard2.0</TargetFrameworks>

```

Comment:
Isn't 'net5.0' backward compatible with 'netstandard2.0' ? See no reason to add another target unless necessarily.